.github/workflows/on_pull_request.yml

```diff
@@ -6,66 +6,18 @@ on:
 
 jobs:
   build:
-    name: Build system packages
+    name: Build system packages and type check
```

Comment:
Type check needs to come with build because type checking website as an example requires built icon types.

I'm still not convinced that each package needs to be type checked after build, becuase to compile the types in build, it is type checked. 

But we do need to type check somethings, like website and theme designer